app/main.js

```diff
@@ -35,8 +41,38 @@ function createWindow () {
   mainWindow.on("closed", function () {
     mainWindow = null
   })
+
+  ipcMain.on("initialized", () => {
+    if (deepLinkURLOnReady != null) {
+      // If open-url event was fired before app was ready
+      loadPopulatePage(deepLinkURLOnReady)
```

Comment:
Loads populate page with deep link if it was triggered before the app was ready (happens when the app isn't open and a deep link is clicked)